core/trino-main/src/main/java/io/trino/execution/scheduler/UniformNodeSelector.java

```diff
@@ -94,11 +100,12 @@ public UniformNodeSelector(
         this.nodeMap = new AtomicReference<>(nodeMap);
         this.minCandidates = minCandidates;
         this.maxSplitsWeightPerNode = maxSplitsWeightPerNode;
-        this.maxPendingSplitsWeightPerTask = maxPendingSplitsWeightPerTask;
+        this.minPendingSplitsWeightPerTask = minPendingSplitsWeightPerTask;
```

Comment:
add `checkArgument` that `minPendingSplitsWeightPerTask <= maxPendingSplitsWeightPerTask`

Reply:
This check is done in `TopologyAwareNodeSelectorFactory` similar to `maxSplitsPerNode >= minPendingSplitsPerTask`

Reply:
`TopologyAwareNodeSelectorFactory` path wont be used when uniform scheduler is used

Reply:
Sorry, I meant `UniformNodeSelectorFactory`.